modules/sdk-core/src/bitgo/staking/iStakingWallet.ts

```diff
@@ -51,13 +51,30 @@ export interface StakeOptions {
    */
   blsSignature?: string;
   /**
-   * coin sepcific staking subtype
+   * stx coin specific staking subtype
    */
   subType?: string;
   /**
-   * btc reward address
+   * stx btc reward address
    */
   btcRewardAddress?: string;
+
+  /**
+   * stx signer pub
+   */
+  signerPub?: string;
+
+  // TODO: remove support to this contract version after STX fork
+  // https://bitgoinc.atlassian.net/browse/EA-3482
+  /**
+   * pox-contract name (valid values are pox-3 and pox-4)
+   */
+  contractName?: 'pox-3' | 'pox-4';
+
+  /**
+   * stx signer signature
+   */
+  signerSignature?: string;
```

Comment:
could we also update interface docs and also put `signerPub` and `signerSignature` next to each other?